source/tutorial/ex5.rst

```diff
@@ -0,0 +1,51 @@
+.. _Exercise 5:
+
+Exercise 5 \\- \ Feature Customization - Splash
+---------------
+
+.. note::
+  **In this exercise, we'll learn how to create display splash screen.**
```

Comment:
This note is a little misleading. The user is not *creating* a splash screen, but simply enabling Slim Bootloader to display the splash screen. The user can customize the splash screen by changing the bitmap image found in the source.

Reply:
fixed